setup/lint.py

```diff
@@ -0,0 +1,12 @@
+"""Modules"""
+import subprocess
```

Comment:
We need all the python files in the repo linted. In hindsight, this should be included as a GitHub action. The Action must fail if the linting fails. 

At the moment, the code appears to only check setup.py which is insufficient. Let's discuss this after all the other fixes are done. This part of the code should be included in a separate issue.

Reply:
@palisadoes, 
I have generated a ```.pylintrc``` file using ```pylint```, and it contains the configurations of code formatting for Python files. The configs can be customized as per our need.
I have integrated the command for linting within the  ```lint``` script in ```package.json```. It would lint the entire project, and exclude some specific folders mentioned in the ```.pylintrc``` file, at ```line 23```. Upon execution, it would give the following output:
![image](https://user-images.githubusercontent.com/75155230/145634328-918abaaa-f9b8-4897-a2d7-47815241a94d.png)

I also wanted to know :
1. Whether I should create a GitHub action for the lint check, 
2. If above point is yes, then should it be another issue
3. The [```lint.py```](https://github.com/kailash360/talawa-api/blob/kailash360/setup/lint.py) file is of no use now. Should it be removed?


I have pushed the code. Do suggest changes, if any.